app/serialised_models.py

```diff
@@ -78,10 +78,10 @@ class SerialisedService(SerialisedModel):
         "name",
         "active",
         "contact_link",
+        # TODO: do we still need this?
         "normalised_service_name",
```

Comment:
I think this will already be missing in the case where the admin app has populated the cache